### ISSUE TITLE
Make Calculate button accessible

### ISSUE THREAD

**Issue by VKTB** [ral-facilities/datagateway#492]
**Description**:
While using the preprod version in dark mode I noticed that the _Calculate_ button in DLS's dataview is difficult to see and therefore fails accessibility. The contrast is very low between the button text and its background colour, see screenshot below.
![image](https://user-images.githubusercontent.com/45173816/100257382-c3d5a080-2f3d-11eb-9476-9b00272b4d7b.png)

**Acceptance criteria**:
- [ ] _Calculate_ button is easy to see and passes accessibility.
